src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveStoreService.kt

```diff
@@ -40,19 +40,6 @@ class IncentiveStoreService(
     nextReviewDateUpdaterService.update(remainingBookingId)
   }
 
-  suspend fun deleteIncentiveReview(prisonerIepLevel: PrisonerIepLevel) {
```

Comment:
i get that we only exposed this for the special sync endpoint, but i do think there's a use case for deleting reviews.

what do you think about resurrecting this as a _new_ api like `DELETE /iep/reviews/booking/{bookingId}/id/{id}`? this method in particular has some business logic that's important to remember for the future (i.e. that there must be a "current" review)

Reply:
don't mean it should be in this pr

Reply:
I noticed these operation are now no longer possible. Deletion of an Incentive record always seemed iffy (at the very least I'd prefer a soft-delete), updating may be more useful (although when I checked these endpoints weren't used much).

However it's definitely out of scope. Plus even if adding an API it wouldn't still be possible at the moment. Which makes me think YAGNI.